hbase10/src/main/java/com/yahoo/ycsb/db/HBaseClient10.java

```diff
@@ -225,7 +224,7 @@ public void cleanup() throws DBException {
 
   public void getHTable(String table) throws IOException {
     final TableName tName = TableName.valueOf(table);
-    synchronized (CONNECTION_LOCK) {
+    synchronized (TABLE_LOCK) {
```

Comment:
#701 asked for this lock to be removed, if I am reading it correctly.

Reply:
He did mention it but I need confirmation from an HBase dev that the table fetch is thread safe in 1.X. I think it wasn't in the past.

Reply:
Connection objects in hbase 1.y are threadsafe, so getTable is fine. The returned Table object is not threadsafe, but making them is supposed to be lightweight and done per-thread when needed.